src/utils/index.js

```diff
@@ -60,3 +60,18 @@ export const changed = (props, objectA, objectB) => {
     objectA && objectB && props.some((prop) => objectA[prop] !== objectB[prop])
   );
 };
+
+const { escapeRegExp } = utils;
+/**
+ * Check whether a piece of text matches the searchvalue, true if no search value given
```

Comment:
`search value` is two words and you don't need the `, true if no search value given` in the top line since you specify that in the return line.

```suggestion
 * Check whether a piece of text matches the search value
```